proto/wippersnapper/i2c/v1/i2c.proto

```diff
@@ -20,188 +21,119 @@ enum BusResponse {
 }
 
 /**
-* I2CBusInitRequest represents a request to
+* I2CBusInit represents a  to
 * initialize the I2C bus from the broker.
 */
-message I2CBusInitRequest {
+message I2CBusInit {
   int32  i2c_pin_scl     = 1; /** The desired I2C SCL pin. */
   int32  i2c_pin_sda     = 2; /** The desired I2C SDA pin. */
   uint32 i2c_frequency   = 3; /** The desired I2C SCL frequency, in Hz. Default is 100000Hz. */
   int32  i2c_port_number = 4; /** The I2C port number. */
 }
 
 /**
-* I2CBusInitResponse represents a response to I2CBusInitRequest
+* I2CBusInitResponse represents a response to I2CBusInit
 */
 message I2CBusInitResponse {
   bool is_initialized  = 1 [deprecated = true, (nanopb).type = FT_IGNORE]; /** True if the I2C port has been initialized successfully, False otherwise. */
   BusResponse bus_response = 2; /** Whether the I2C bus initialized properly or failed. **/
 }
 
-/**
-* I2CBusSetFrequency represents a request to change the
-* I2C clock speed to a desired frequency, in Hz.
-*/
-message I2CBusSetFrequency {
-  uint32 frequency = 1; /** The desired I2C SCL frequency, in Hz. */
-  int32 bus_id     = 2; /** An optional I2C bus identifier, if multiple exist. */
-}
 
 /**
-* I2CBusScanRequest represents the parameters required to execute
+* I2CScan represents the parameters required to execute
 * a device's I2C scan.
 */
-message I2CBusScanRequest {
+message I2CScan {
   int32 i2c_port_number              = 1; /** The desired I2C port to scan. */
-  I2CBusInitRequest bus_init_request = 2; /** The I2C bus initialization request. */
+  I2CBusInit bus_init = 2; /** The I2C bus initialization . */
 }
 
 /**
-* I2CBusScanResponse represents a list of I2C addresses
-* found on the bus after I2CBusScanRequest has executed.
+* I2CScanResponse represents a list of I2C addresses
+* found on the bus after I2CScan has executed.
 */
-message I2CBusScanResponse {
+message I2CScanResponse {
   repeated uint32 addresses_found  = 1 [packed=true, (nanopb).max_count = 120]; /** The 7-bit addresses of the I2C devices found on the bus, empty if not found. */
   BusResponse bus_response         = 2; /** The I2C bus' status. **/
 }
 
 /**
-* I2CDeviceSensorProperties contains
+* I2CSensorProperties contains
 * the properties of an I2C device's sensor such as
 * its type and period.
 */
-message I2CDeviceSensorProperties {
-  SensorType sensor_type = 1;
-  uint32 sensor_period   = 2;
+message I2CSensorProperties {
+  wippersnapper.sensor.v1.SensorType sensor_type = 1;
 }
 
 
 /**
-* Represents a list of I2CDeviceInitRequest messages.
+* Represents a list of I2CInit messages.
 */
-message I2CDeviceInitRequests {
-  repeated I2CDeviceInitRequest list = 1;
+message I2CInits {
+  repeated I2CInit list = 1;
 }
```

Comment:
I'm curious if we might want to drop this and send multiple `I2CInit` requests, which is how it works with DS18X20. We only use this repeated one during boot to send every I2C component across all addresses, seems like it might even be a payload size risk if a user loads up a dozen+ components across a few addresses?

Reply:
> might even be a payload size risk if a user loads up a dozen+ components across a few addresses?

This is a legitimate concern. I think we can send multiple I2CInit requests, it's not a problem and I believe this was a temp. fix for when we had a MQTT client bug.